recipes/genblasta/meta.yaml

```diff
@@ -0,0 +1,23 @@
+about:
+  home: http://genome.sfu.ca/genblast/download.html
+  license: 'GNU'
+  summary: 'genBlast is a program suite, consisting of two programs: genBlastA and genBlastG.
+      genBlastA parses local alignments, or high-scoring segment pairs (HSPs)
+      produced by local sequence alignment programs such as BLAST and WU-BLAST and identify groups of HSPs.'
+
+build:
+  number: 0
+  skip: True # [osx]
+
+package:
+  name: genblasta
+  version: '1.0.4'
+
+requirements:
+  run:
```

Comment:
@jerowe the problem with precompiled binaries is that you often don't know how they where produced. bioconda and conda-forge are trying hard to have one unified build infrastructure based on a as old as possible cent-os. As a result these binaries are relocatable to any other OS, which is not true if you compile them on a recent Ubuntu for example.
So we very much favour compiling from source if possible.